information.txt

```diff
@@ -24,18 +24,22 @@ Overall evaluation: 0 (borderline paper)
 The concept is interesting, however the major revision of the paper is necessary for making it ready for presentation at the conference.
 
+<<<CHRIS NOTE - I feel like most of this reviewer's comments should be addressed in the introduction. They are not details / implementation questions, but more "motivation and why is this useful?" questions. I tried
+to separate out the intro into a shorter "proper" intro section, and a second more long-winded one, to make
+these points more clearly >>>
+
 1) I have a feeling that in fact the authors would like to compare the clouds with traditional work with repositories, but I cannot see any significant defferences amoung those two tevhnologies in the paper. The authors use the cloud mainly to give for the students the remote access to the teaching material and data and an opportunity to upload their work into cloud.
-It is unclear for me what kind of other services are used.
+It is unclear for me what kind of other services are used.  <--- emphasized the fact that students don't need to download anything new onto their computer, they only go to a website.
 
-2) The author mention the requirement of Knowledge in cloud-computing infrastructure as the drawback of the cloud approach. I cannot understand it. The main aim of using the cloud is to make such infrastructure 'invisible' to the end users (students). Why do they need such knowledge?
+2) The author mention the requirement of Knowledge in cloud-computing infrastructure as the drawback of the cloud approach. I cannot understand it. The main aim of using the cloud is to make such infrastructure 'invisible' to the end users (students). Why do they need such knowledge?  <--- Tried to clarify language in the intro saying only one person needs any knowledge of cloud infrastructure.
 
-3) The conclusions section should be moved to the end of the paper
+3) The conclusions section should be moved to the end of the paper  <--- I kinda wanna push back on this. I like the methods being last because it's more of an appendix. what do people think?
 
-4) I expected some experimental verification and analysis of the compared teaching methodologies.. for the concrete workshop..
+4) I expected some experimental verification and analysis of the compared teaching methodologies.. for the concrete workshop..  <--- added language that this was a case study to the intro
 
-5) All figures but Fig 2 are too small and unreadable
+5) All figures but Fig 2 are too small and unreadable  <--- TODO still need to address this.
 
-6) The motivation of using XSEDE Jetstream cloud instead of OpenStack itself is not clear for me.
+6) The motivation of using XSEDE Jetstream cloud instead of OpenStack itself is not clear for me. <--- I added a paragraph mentioning that the XSEDE cluster is widely available and that really you only need a service that runs Kubernetes.
```

Comment:
ok I added this in and tried to make it more clear that it's not XSEDE-specific. See the bulleted list in the methods section for example. Perhaps a better way to do it would be to color certain points gray if they're XSEDE-specific, and leave more general to-do items in black?